certified-connectors/Ideanote/README.md

```diff
@@ -0,0 +1,34 @@
+## Ideanote Connector
+
+Ideanote is the next-generation innovation software that has everything you need to collect, develop, prioritize and act on more of the right ideas.
+
```

Comment:
@runem Please find a template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) and add the other sections.

Reply:
@runem Please add the required sections such as "Publisher". I have already shared the template in my previous comment.

Reply:
@sriyen-msft Sorry, I misread the publisher section as being only required for independent publishers. The section has been added now.